pyodide_build/pywasmcross.py

```diff
@@ -59,10 +59,43 @@ def collect_args(basename):
         path = path.replace(str(ROOTDIR) + ':', '')
     env['PATH'] = path
 
+    # determine the package name from the current directory
+    re_res = re.match('.*packages/(?P<name>[\w\-]+)/build.*',
+                      env['PWD'])
+    if re_res:
+        package_name = re_res.group('name')
+    else:
+        package_name = None
+
+    # Skip compilations of C/Fortran extensions for the target environement.
+    # We still need to generate the output files for distutils to continue
+    # the build.
+    # TODO: This may need slight tuning for new projects. In particular,
+    #       currently ar is not skipped, so a known failure would happen when
+    #       we create some object files (that are empty as gcc is skipped), on
+    #       which we run the actual ar command.
```

Comment:
I think -- but I have not actually tried it, and I'm not sure if `ar` is used in any of the C extensions.

Reply:
I'm pretty sure Numpy uses it, IIRC.  But maybe none of the others do.

Reply:
Yes.  And if we run into trouble in the future, it should be straightforward to write a skipping version of ar that generates an empty `.a` file as well.